python-crane.spec

```diff
@@ -63,19 +65,22 @@ rm -rf %{buildroot}%{python2_sitelib}/tests
 %{python2_sitelib}/crane
 %{python2_sitelib}/crane*.egg-info
 %{_usr}/share/crane/
+%{_libexecdir}/crane/
 %dir %{_var}/lib/crane/
 %dir %{_var}/lib/crane/metadata/
 %doc AUTHORS COPYRIGHT LICENSE README.rst
 
 
 %post
-semanage fcontext -a -t httpd_sys_content_t '%{_var}/lib/crane(/.*)?'
-restorecon -R -v %{_var}/lib/crane
+if /usr/sbin/selinuxenabled; then
```

Comment:
Switching these to be more explicit would be good. Using the `if [ $? -eq 0 ]` would be good.